manifests/nodejs.yml

```diff
@@ -51,7 +51,11 @@ tests/:
         Test_API_Security_RC_ASM_DD_scanners: *ref_5_3_0
         Test_API_Security_RC_ASM_processor_overrides_and_custom_scanner: irrelevant (waf does not support it yet)
       test_apisec_sampling.py:
-        Test_API_Security_sampling: *ref_4_21_0
+        Test_API_Security_Sampling_Rate: *ref_4_21_0
```

Comment:
This will be updated to `missing_feature` after the Node tracer is released with the new sampling feature.